stations.csv

```diff
@@ -22860,8 +22860,8 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 24416;Selgua;selgua;7178300;;41.9061;0.1484;;f;ES;f;Europe/Madrid;t;;;f;;f;;f;;f;;f;;f;;f;;f;78300;t;f;;;;;;;;;;;;;;;;;;
 24417;Serin;serin;7115302;;43.504815;-5.782733;;f;ES;f;Europe/Madrid;t;;;f;;f;7115302;f;;f;;f;;f;;f;;f;15302;t;f;;;;;;;;;;;;;;;;;;
 24418;Setenil;setenil;7155005;;36.863306;-5.12155;;f;ES;f;Europe/Madrid;t;;;f;;f;7155005;f;;f;;f;;f;;f;;f;55005;t;f;;;;;;;;;;;;;;;;;;
-24419;Sevilla Santa Justa;sevilla-santa-justa;7151003;;37.391925;-5.975264;;f;ES;f;Europe/Madrid;t;ESAGN;;f;;f;7100180;f;;f;;f;;f;;f;;f;51003;t;f;;;Seville;;Séville;Siviglia;;;;;;;Sewilla;Sevilha;Севилья;;;
-24420;Sevilla-Virgen del Roció;sevilla-virgen-del-rocio;7151110;;37.362827;-5.975831;;f;ES;f;Europe/Madrid;t;;;f;;f;7151110;f;;f;;f;;f;;f;;f;51110;t;f;;;Seville;;Séville;Siviglia;;;;;;;Sewilla;Sevilha;Севилья;;;
+24419;Sevilla Santa Justa;sevilla-santa-justa;7151003;;37.391925;-5.975264;6683;f;ES;f;Europe/Madrid;t;ESAGN;;f;;f;7100180;f;;f;;f;;f;;f;;f;51003;t;f;;;Seville;;Séville;Siviglia;;;;;;;Sewilla;Sevilha;Севилья;;;
```

Comment:
Sevilla Santa Justa is now marked as main station.

Commit message has been rewritten too.